shorebird_code_push/example/lib/main.dart

```diff
@@ -1,125 +1,108 @@
 import 'package:flutter/material.dart';
 
+import 'package:shorebird_code_push/shorebird_code_push.dart';
+
+final ShorebirdCodePush shorebirdCodePush = ShorebirdCodePush();
```

Comment:
does this need to be a global variable? Could it be initialized as part of the `StatefulWidget` `State`?

Reply:
Good call, done